docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,163 @@
+-   Pulse associates all the Bitbucket user groups of the author of a merged pull request with the corresponding deployment, excluding the user groups with less than two members. Pulse only takes the changes to Bitbucket user groups into account on pull requests merged after those changes.
+
+## Collected data
+
+The table below lists the data that the Bitbucket integration collects from your Bitbucket workspace, together with:
+
+-   The mapping between the data collected from Bitbucket and the [Pulse data model](https://ingestion.pulse.codacy.com/v1/api-docs#tocs_event)
+-   The metrics that Pulse calculates from the data to display on the dashboards
+
+<table>
+<thead>
+<tr>
+<th><strong>Data collected from Bitbucket</strong></th>
+<th><strong>Mapping to Pulse data model</strong></th>
+<th><strong>Used in</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+    <td>Pull request commits</td>
+    <td>
+        <p>Changes:</p>
+        <ul>
+            <li><code>change_id</code>: commit UUID</li>
+            <li><code>time_created</code>: commit author date<sup><a href="#commit-author-date">1</a></sup></li>
+            <li><code>system</code>: repository name</li>
+        </ul>
+    </td>
+    <td>Lead time for changes on the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a></td>
+</tr>
+<tr>
+    <td>Pull requests</td>
+    <td>
+        <p>Deployments:</p>
+        <ul>
+            <li><code>deploy_id</code>: pull request ID</li>
+            <li><code>system</code>: repository name</li>
+        </ul>
+    </td>
+    <td>Deployment frequency and Change failure rate on the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a></td>
+</tr>
+<tr>
+    <td>User groups</td>
+    <td>
+        <p>Deployments:</p>
+        <ul>
+            <li><code>teams</code>: Bitbucket user groups of the author of a deployment<sup><a href="#deployment-teams">2</a></sup></li>
+        </ul>
+    </td>
+    <td>Filters the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a> by the contributions made by the team</td>
+</tr>
+</table>
+
+<sup><span id="commit-author-date">1</span></sup>: Pulse uses the commit author's date since it is more accurate. The committer date can be changed (e.g.: fast-forward) and stop reflecting the real creation date of the change.
+
+<sup><span id="deployment-teams">2</span></sup>: Adding a new user group or changing the composition of a user group on Bitbucket only affects new data from that moment onwards and doesn't have an immediate impact on the dashboards. Also, deleted user groups on Bitbucket will continue to show in Pulse.
```

Comment:
This one is a bit more debatable, but in the spirit of readability and clarity about this being a current feature, we may want to use the present tense here as well.

```suggestion
<sup><span id="deployment-teams">2</span></sup>: Adding a new user group or changing the composition of a user group on Bitbucket only affects new data from that moment onwards and doesn't have an immediate impact on the dashboards. Also, deleted user groups on Bitbucket are still visible in Pulse.
```